docs/best-practices/log-container-sidecarset.md

```diff
@@ -91,22 +91,19 @@ metadata:
   name: filebeat-sidecarset
 spec:
   selector:
-    # Pod labels that need to be injected into the sidecar container
```

Comment:
there is also indentation problem in L33, around the line 
 ### K8S Sidecar Model Disadvantage

in addition, please add the resource declaration in the code blocks(add apiserver, kind part)